src/quo/components/profile/expanded_collectible/view.cljs

```diff
@@ -21,9 +20,9 @@
       :value           counter}]))
 
 (defn- fallback-view
-  [{:keys [label theme counter]}]
-  [rn/view
-   {:style (style/fallback {:theme theme})}
+  [{:keys [label theme counter on-mount]}]
```

Comment:
rather than `on-mount` how about calling it something more expressive? 
We know the behaviour is `on-mount` anyway because of the hook name.
e.g `on-collectible-load` is more descriptive
`(rn/use-mount on-collectible-load)`

Reply:
It's a placeholder and I'd like to execute some code `on-mount` :thinking:  so I think it's a good name.

It's being called as:
```clojure
{:on-mount on-collectible-load
 ,,,}
```

Do you prefer it'd be:
```clojure
{:on-collectible-load on-collectible-load
 ,,,}
```
even though this is a placeholder without a collectible? :thinking: